### ISSUE TITLE
GitHub actions version always a version behind if explicitly setting version

### ISSUE THREAD

**Issue by tunetheweb** [github/super-linter#1839]
## Describe the bug

Saw the release note about SQLFluff in 4.6.0 and was keen to try it, but even after upgrading it still used 4.5.1:

![GitHub Action screenshot](https://user-images.githubusercontent.com/10931297/128579350-3247df07-4b8e-4d02-a76e-806d002db11f.png)

## Expected behavior

Should use 4.6.0 when that's what you tell it to use.

## Steps to Reproduce

Set up an action with this config (not explicitly give the version rather than `latest`):

```
uses: github/super-linter@v4.6.0
```

Run it and notice it uses 4.5.2. You can force it to use 4.6.0 instead by using Docker version:

```
uses: docker://github/super-linter:v4.6.0
```

But then it's not automatically kept up tot update by dependabot.

## Logs

Here's an example run: https://github.com/HTTPArchive/almanac.httparchive.org/runs/3266589286?check_suite_focus=true

**Comment by admiralAwkbar** [comment]
@tunetheweb Hey thanks for the catch! not sure why versioning has been such an issue with this... Will try to debug some more

**Comment by tunetheweb** [comment]
Solved now (see #1913 ). Closing.
